nvaccess/nvda
# replace controlTypes.STATE_* with controlTypes.State.*

### Link to issue number:

Part of #12549 

### Summary of the issue:

`controlTypes.STATE_*` are deprecated, to be replaced by `controlTypes.State.*`

### Description of how this pull request fixes the issue:

* replaces all instances of `controlTypes.STATE_` with `controlTypes.State.` across .py files
* search and manually replace final STATE_ constant usages 
     * Searched using the regex `[^\.]\bSTATE_`, which finds STATE_ instances
where:
          * it is not of the form `<Foo>.STATE_ `(as oleacc.STATE_* exists)
          * STATE_ is the prefix of the symbol (as IA2_STATE_* exists)

### Testing strategy:

Manual testing
* Confirm the above steps will not introduce issues, and cover all cases.
* Reproduce those steps and perform a diff with this branch

Existing unit tests exist for controlTypes.
New unit tests are written to ensure displayString and negativeDisplayString works

Existing system tests will work as a basic smoke test for this core feature

### Known issues with pull request:

The lint check is likely to fail

### Change log entries:

None needed

### Code Review Checklist:



  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
